src/Views/Form/Templates/Classic/resources/css/_receipt.scss

```diff
@@ -0,0 +1,204 @@
+.give-receipt-classic {
```

Comment:
I think I’ll add a task for myself, when I go through and optimize all the styles, to make these styles consistent with everything else (i.e. using less specific selectors, `rem` and `em` units, etc.). These are good for now.